### DISCUSSION TITLE
Jupyterlab notification overflow

### DISCUSSION THREAD

**Discussion by grallewellyn** [jupyterlab/jupyterlab#14760] [Show and tell]
When we upgraded jupyterlab to use the built in notifications, we were still having this error:
<img width="654" alt="Screenshot 2023-06-26 at 1 16 10 PM" src="https://github.com/jupyterlab/jupyterlab/assets/114033465/3505c7b1-5b9e-464d-8623-8bb1927a0742">
where longer notifications block the delete button 

If you add 
```
.Toastify__toast-body {
    overflow: auto;
  }
```
to your style.css in your project, this resolves this issue

**Comment by jtpio** [comment]
Thanks @grallewellyn.

Is this issue still happening with the latest version of JupyterLab 4?

If so, would you like to open a PR to add the missing `overflow`? Thanks!
